server/schedule/region_scatterer_test.go

```diff
@@ -339,5 +348,22 @@ func (s *testScatterRegionSuite) TestScatterGroup(c *C) {
 			c.Assert(max, GreaterEqual, uint64(20))
 			c.Assert(max-min, LessEqual, uint64(3))
 		}
+		cancel()
 	}
 }
+
+func (s *testScatterRegionSuite) TestSelectedStoreGC(c *C) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	stores := newSelectedStores(ctx, true)
+	stores.put(1, "testgroup")
+	_, ok := stores.getStore("testgroup")
+	c.Assert(ok, Equals, true)
+	_, ok = stores.getGroupDistribution("testgroup")
+	c.Assert(ok, Equals, true)
+	time.Sleep(gcTTL)
```

Comment:
I don't know whether it is a good idea to use `Sleep` during the test, it might make the tests run slower.
Any better suggestions?

Reply:
sleep here is ok.

Reply:
Can we change the gcTTL to shorten the sleep time?

Reply:
> Can we change the gcTTL to shorten the sleep time?

Maybe we do can make gcTTL a variable. Except the convenience in testing, we may also expost it to the config in the future.